homeassistant/components/rest/entity.py

```diff
@@ -15,29 +17,20 @@ class RestEntity(Entity):
 
     def __init__(
         self,
-        coordinator: DataUpdateCoordinator[Any],
+        coordinator: DataUpdateCoordinator[Any] | None,
         rest: RestData,
-        resource_template,
-        force_update,
+        resource_template: Template | None,
+        force_update: bool,
     ) -> None:
         """Create the entity that may have a coordinator."""
         self.coordinator = coordinator
```

Comment:
The `coordinator` is only accessed in the base class / `RestEntity`. To make sure it isn't assigned later, it could make sense to prefix it with an underscore.

Reply:
done